libraries/AP_PiccoloCAN/AP_PiccoloCAN.h

```diff
@@ -125,6 +136,10 @@ class AP_PiccoloCAN : public AP_CANDriver
 
     } _esc_info[PICCOLO_CAN_MAX_NUM_ESC];
 
+    // Piccolo CAN parameters
+    AP_Int32 _esc_bm;       //! ESC selection bitmask
+    AP_Int32 _esc_hz;       //! ESC update rate (Hz)
```

Comment:
I think 16 bit will do for the rate in Hz ?

Reply:
Done